arpa-exporter/src/lib/logging.py

```diff
@@ -23,7 +23,7 @@
 ]
 
 processors: List[Callable] = shared_processors + []
-if sys.stderr.isatty():
+if sys.stderr.isatty():  # pragma: nocover
```

Comment:
Unrelated to the bug, I just noticed this was throwing off coverage metrics, but this expression will never evaluate differently in a realistic unit-testing scenario.